frontend/src/container/GridCardLayout/GridCard/FullView/index.tsx

```diff
@@ -193,13 +196,17 @@ function FullView({
 
 			<div
 				className={
-					isDashboardLocked ? 'graph-container disabled' : 'graph-container'
+					isDashboardLocked
```

Comment:
use classnames to make it more readable.

Reply:
Done ✅